saleor/order/emails.py

```diff
@@ -44,9 +44,11 @@ def collect_data_for_email(
     order_pk: int, template: str, redirect_url: str = ""
 ) -> dict:
     """Collect the required data for sending emails."""
-    order = Order.objects.prefetch_related("lines__variant__product__images").get(
-        pk=order_pk
-    )
+    order = Order.objects.prefetch_related(
+        "lines",
+        "lines__variant",
+        "lines__variant__product",
```

Comment:
No need to provide each of them separately. Previously it was correct. Also, I don't think that we should remove the images prefetch from here.
![image](https://user-images.githubusercontent.com/23282005/107200876-8e970580-69f8-11eb-9971-f15613821467.png)